LICENSE.txt

```diff
@@ -1,91 +1,270 @@
-==============================================================================
-compiler-builtins License
-==============================================================================
+compiler-builtins as a whole is available for use under both the MIT license
+and the Apache-2.0 license with the LLVM exception (MIT AND Apache-2.0 WITH
+LLVM-exception).
 
-The compiler-builtins crate is dual licensed under both the University of
-Illinois "BSD-Like" license and the MIT license.  As a user of this code you may
-choose to use it under either license.  As a contributor, you agree to allow
-your code to be used under both.
+As a contributor, you agree that your code can be used under either the MIT
+license, or the Apache-2.0 license, or the Apache-2.0 license with the LLVM
+exception.
```

Comment:
To be clear, this means I agree that you can choose any of these licenses, right? It doesn't mean I as the contributor can choose one of these licenses to make the contributor under? IMO this is not clear enough.